rclrs/src/node/publisher.rs

```diff
@@ -107,6 +108,30 @@ where
         })
     }
 
+    /// Returns the topic that the publisher is publishing data to
+    pub fn get_topic(&self) -> Result<String, RclrsError> {
+
+        let mut _topic_name: String = String::default();
+        // SAFETY: No preconditions for the functions called.
+        // Edge cases for all unsafe variables created have been handled
+        // The unsafe variables created get dropped as soon as they're used
+        unsafe {
+            let raw_topic_pointer = rcl_publisher_get_topic_name((&mut *self.handle.lock()) as *mut _);
+            match CStr::from_ptr(raw_topic_pointer).to_str() {
```

Comment:
In other places I've just used `to_string_lossy().into_owned()`, which also has the advantage that the function can just return a `String`. Would be nice to do that here too for consistency.

Reply:
noted, these changes have been made to add consistency